dts/bindings/led_strip/worldsemi,ws2812-gpio.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2019, Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+title: Worldsemi WS2812 LED strip, GPIO binding
+
+description: |
+  Driver bindings for bit-banging a WS2812 or compatible LED strip.
+
+  The GPIO driver uses inline assembly, and isn't available for all
+  boards.
+
+compatible: "worldsemi,ws2812-gpio"
+
+include: [base.yaml, ws2812.yaml]
+
+properties:
+  label:
```

Comment:
This property should be likely moved to `dts/bindings/led_strip/ws2812.yaml` as it is not specific to ws2812-gpio. The `worldsemi,ws2812-spi.yaml` doesn't specify it. However, the relevant define is generated also for the spi version of the driver.

Reply:
The SPI yaml doesn't need this because spi-device.yaml (which it includes) already has this line.

Since the gpio binding inherits from base.yaml instead, it's needed here.